Extension/src/common.ts

```diff
@@ -951,7 +966,7 @@ function extractArgs(argsString: string): string[] {
 }
 
 export function extractCompilerPathAndArgs(inputCompilerPath?: string, inputCompilerArgs?: string[]): CompilerPathAndArgs {
-    let compilerPath: string | undefined = inputCompilerPath;
+    let compilerPath: string | undefined = inputCompilerPath?.toLowerCase();
```

Comment:
I think converting the path to lower case here would cause some issues.  The compiler path is only case insensitive on Windows.  And, at this point, it could contain compiler args as well.  This function parses out those args, and adds them to `additionalArgs`.  Those casing of compiler args matter on all platforms.